example/spec/helpers/testUtils.js

```diff
@@ -89,6 +89,16 @@ function loadConfigFromKes(type) {
 const loadConfigFromYml = () => {
   const config = loadYmlFile('./config.yml');
```

Comment:
This could all be replaced by:

```js
const config = {
    stackName: process.env.DEPLOYMENT,
    sqs_consumer_rate: 300,
    ...loadYmlFile('./config.yml')
  };
```

Reply:
After looking into it, it looks like we are losing the ability to set the sqs consumer rate in the terraform deployment. I recall this being a specific feature request in the past. I'd like to restore that if possible.

Reply:
Filed CUMULUS-1625.